scripts/load_data_to_sqlite.py

```
@@ -0,0 +1,65 @@
+                    ruling_type=ruling.ruling_type,
+                    question=ruling.question,
+                    answer=ruling.answer,
+                    text=ruling.text,
+                    provenance=ruling.provenance.model_dump(),
+                    original_html_snippet=ruling.original_html_snippet,
+                    tags=ruling.tags,
+                )
+                session.add(ruling_db)
+            except Exception as e:
+                logging.error(f"Error processing ruling: {ruling_dict.get('id')}, error: {e}")
+
```

Ruff (0.11.9)

`error()` call on root logger

(LOG015)